pandas/tests/groupby/test_transform.py

```diff
@@ -911,6 +911,36 @@ def test_pct_change(test_series, freq, periods, fill_method, limit):
         tm.assert_frame_equal(result, expected.to_frame("vals"))
 
 
+def test_ffill_non_unique_multilevel():
```

Comment:
Is there an easy way to parametrize on ffill / bfill? Seems like testing both would be nice

Reply:
Agreed - now testing both ffill and bfill.